conf/ccga.config

```diff
@@ -0,0 +1,72 @@
+/*
+vim: syntax=groovy
```

Comment:
You can remove these two lines - not needed any more since GitHub added syntax highlighting for nextflow code.